src/mixins/filing-template-mixin.ts

```diff
@@ -971,9 +971,11 @@ export default class FilingTemplateMixin extends DateMixin {
 
     // store Approval Type
     if (filing.restoration.approvalType) {
+      // get approval type from draft
       this.setRestorationApprovalType(filing.restoration.approvalType)
-    } else {
-      this.setRestorationApprovalType(this.getStateFilingRestoration?.approvalType)
+    } else if (this.getStateFilingRestoration?.approvalType) {
+      // get approval type from state filing
+      this.setRestorationApprovalType(this.getStateFilingRestoration.approvalType)
```

Comment:
This is a safety check only (because the previous code could have stored "undefined" as the restoration approval type, which could cause reactivity issues).

This is not the fix for the bug in the subject ticket/PR.